Jenkinsfile

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env groovy
+
+node('docker && awsaccess') {
+  cleanWorkspace()
+
+  stage('Build docker image') {
+    checkout scm
+    def imageId = 'docker.io/resolvme:latest'
+    dockerImage = docker.build('resolvme:latest')
+    dockerTestImage = docker.build('resolvme:jenkins', "-f Dockerfile-dev .")
+  }
+
+  dockerTestImage.inside() {
+    stage('Run linter') {
+      sh 'bundle exec rubocop --cache false bin lib spec || true'
```

Comment:
This line doesn't check Gemfile, .gemspec, or Rakefile

Also, this won't fail if there is a syntax error. Use the `--fail-level F` argument to do that and still keep succeeding if there are any other issues.